node/node_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"os"
 	"path/filepath"
 	"strconv"
+	"strings"
```

Comment:
Not the right spot but... 

```suggestion
	"runtime"
	"strings"
```